EZSwiftExtensionsTests/ArrayTests.swift

```diff
@@ -285,6 +292,54 @@ class ArrayTests: XCTestCase {
         XCTAssertEqual(duplicatedElementsArrayOrderMaintained.unique(), [2, 1, 3])
     }
     
+    func testStep() {
+        XCTAssertNil(self.numberArray.step(-1))
+        XCTAssertNil(self.numberArray.step(0))
+
+        XCTAssertEqual(self.numberArray.step(2)!.count, 4)
```

Comment:
This is hard to read. Can you make some comments which describe the output of the step just prior to the count and array validations.